doc/manual/manual.tex

```diff
@@ -6310,11 +6310,78 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 parameter (the division by 0.02 in the formula for the smoothed initial
 condition). However, for more exact results, one can choose
 this half width parameter proportional to the mesh size and thereby get more
-accurate results. Since the point of the section was to demonstrate the reason
-for the lack of convergence, but we leave this as an experiment to the reader.
+accurate results. The point of the section was to demonstrate the reason
+for the lack of convergence.
 
+In this section we extend the Van Keken cookbook following up the work previously completed by Cedric Thieulot, Juliane Dannberg,
+Timo Heister and Wolfgang Bangerth.  \textit{This section contributed by Grant Euen, Tahiry Rajaonarison, and Shangxin Liu as part of the Geodynamics and ASPECT class at Virginia Tech.}  
```

Comment:
a "section" is an actual subdivision in a document, but here you only simply continue the current section. Just replace these two lines by the following:
```
  In the following, let us extend these results in a variety of directions. \textit{These extensions were contributed by ...}
```